### PR TITLE
Feature/build out comp

### LINKED ISSUE #17: Build out individual recipe card
- [x] Image on top
- [x] Name below
- [x] Save and favorite buttons at the bottom
- [x] User should be able to click on image or title to navigate to the recipe page

### LINKED ISSUE #23: Switches to recipe view when user clicks on a recipe
- [x] Image on the right
- [x] Title
- [x] Price
- [x] Ingredients
- [x] Directions

### LINKED ISSUE #24: Build 'Navigation Bar'
- [ ] Add link to main view on the left
- [ ] Add link to 'Browse Recipes' view
- [ ] Add link to 'Browse Recipes' view filtered by faves
- [ ] Add link to 'Cookbook' view (saved recipes)
- [ ] Add Search bar on the far right

### LINKED ISSUE #25: Build 'Featured Recipe' display at top of main page
- [x] Large background image that covers the whole page, except the nav bar
- [x] Small box with recipe name
- [x] Box should include 'favorite' and 'save to cookbook' buttons

### PR DESCRIPTION
**Pull request by mr-ryan12**
What (if any) features are you implementing?
Added functionality to get all of the recipes on the main view.

What (if anything) did you refactor?
Refactored some HTML and CSS for readability.

Were there any issues that arose?
Not at the time.

Is there anything that you need from your teammate?
Code review/pull request review.

Any other comments, questions, or concerns?
Not at the moment.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,9 +2,54 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
+    <nav class="nav-bar">
```
Comment (oacortez) [inline src/index.html:9]: Thank you for building out this nav bar looks great!

File: src/index.html [hunk 2]
```diff
@@ -2,9 +2,54 @@
...
+          <option value="side dish">Side Dish</option>
+          <option value="snack">Snack</option>
+          <option value="spread">Spread</option>
+          <option value="starter">Starter</option>
+        </select>
+      </section>
+      <input class="search-bar" id="search-bar" placeholder="Search by name or ingredient">
+    </nav>
+    <main>
+      <section class="recipe-of-the-day">
```
Comment (oacortez) [inline src/index.html:41]: Great section build where it includes a feature recipe

**Review by oacortez — APPROVED** [review]
Thank you for building the set-up for our HTML and CSS and as well including a function, so we can display our recipe cards to the browser.

**Review by philalewis — APPROVED** [review]
I like how it's going so far. We need to build out the recipe view page and do some more styling work. I'm thinking we should put hte browse page seperate form the home page. Looking good so far though.

File: src/index.html (hunk 1 above)
Comment (mr-ryan12) [reply src/index.html:9]: Appreciate it! I think the drop down menu gives the navigation bar a cleaner look and will enhance the user experience by simplifying the filter functionality.

**Pull request merged**